lightwood/encoders/numeric/ts_numeric.py

```diff
@@ -48,23 +48,18 @@ def encode(self, data, extra_data=None):
                     mean = self._abs_mean
                 if real is not None and mean > 0:
                     vector[0] = 1 if real < 0 and not self.positive_domain else 0
-                    vector[1] = math.log(abs(real)) if abs(real) > 0 else -20
```

Comment:
why is this statement no longer necessary? it may be obvious later in the code but not atm

Reply:
Same as above: it was not being used anyway, so I removed it